app/views/showField.blade.php

```diff
@@ -73,18 +74,16 @@
 				<a href="#close" title="Close" class="close">X</a>
 				<h1>MOUNDS OF MOUNDS!</h1>
 				<div id="seed_table">	
-						@if(empty($userItems))
+						@if(!empty($userItems))
 							{{ Form::open(array('action' => array('HomeController@plant', 'method' => 'PUT')))}}
 							@foreach($userItems as $item)
 								@if($item->is_seed)
 									{{{$item->name}}} 
-									{{ Form::radio('seedID', $item->id) }}
+									{{ Form::radio('seedName', $item->name) }}
 									<br>
```

Comment:
The code you made changes to is not live, I'll show you my changes when you get in